packages/lib/src/ui/blocks/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,84 @@
+import { Block } from "../../components/Block/Block"
+import { Container } from "../../components/Container/Container"
+import styled from "@emotion/styled"
+import { Header3, SmallText } from "../../components/Typography/Typography"
+import React from "react"
+import { GalleryProps } from "./galleryDef"
+
+import Rectangle_1 from "../../assets/vectors/potatoes/rectangle_1.svg"
+import Rectangle_2 from "../../assets/vectors/potatoes/rectangle_2.svg"
+import Rectangle_3 from "../../assets/vectors/potatoes/rectangle_3.svg"
+import Rectangle_4 from "../../assets/vectors/potatoes/rectangle_4.svg"
+import { max, theme } from "../../styles/theme"
+
+const rectangles = [Rectangle_1, Rectangle_2, Rectangle_3, Rectangle_4]
+
+export interface GalleryViewProps {
```

Comment:
Tento interface přesuň do `galleryDef`. Je to jediné místo, kde se používá, tak nemá smysl, aby byl tady.